code/game/objects/items/weapons/autolathe_disk/gun_parts_disks.dm

```diff
@@ -127,14 +127,13 @@ Avoid any disks here being found or used commonly beyond an intended purpose, ot
 		/datum/design/autolathe/gun_frame/judiciary,
 		/datum/design/autolathe/gun_frame/colt,
 		/datum/design/autolathe/gun_frame/amnesty,
-		/datum/design/autolathe/gun_frame/freedom,
 		/datum/design/autolathe/gun_frame/judge,
+		/datum/design/autolathe/gun_frame/freedom,
 		/datum/design/autolathe/gun_frame/bastard,
-		/datum/design/autolathe/gun_frame/mamba,
 		/datum/design/autolathe/gun_frame/specop,
-		/datum/design/autolathe/gun_frame/dp,
```

Comment:
DP was always a marshal gun from day one. It serves no function in blackshield when they have the arguably better Heroic. On the other hand it serves a great function for marshals shop as a cheaper LMG they can sell.

Reply:
This now takes it from a cheap low-end LMG to a super niche gun stuck behind a department with no direct access to gun parts or pan mags who will likely never use it or trade them out. :P

Reply:
Marshals can make the takeshi who infact can be converted into heroic, for same price of plasteel and similar gunparts, it's just a gun that serves no purpose at all gonna be real. Bastard LMG (it uses carbine rounds why tf is it called SMG) is way better choice when you want a cheap and lightweight machine gun since it has no slowdown compared to any of the other ones. What I'm thinking is just removing it entirely from the disk but shoving it into the marshal vendor so anyone can buy it but if you want heroic or takeshi you go to the spec.

Reply:
As for the gun parts it uses all things blackshield can acquire, all of those are on lonestar disk, guild can make them, wooden grip MG mechanism and 7.62 barrels are also incredibly common in trash piles and can be acquired from guns they already have (cowboy, mosin, STS BR)